test/domain based tests/claimLosslessReward.js

```diff
@@ -118,23 +117,23 @@ describe(scriptName, () => {
         );
       });
     });
-  });
 
-  describe('when lossless team claims two times', () => {
-    it('should revert', async () => {
-      await env.lssGovernance.connect(adr.lssAdmin).losslessClaim(1);
+    describe('when lossless team claims two times', () => {
+      it('should revert', async () => {
+        await env.lssGovernance.connect(adr.lssAdmin).losslessClaim(1);
 
-      await expect(
-        env.lssGovernance.connect(adr.lssAdmin).losslessClaim(1),
-      ).to.be.revertedWith('LSS: Already claimed');
+        await expect(
+          env.lssGovernance.connect(adr.lssAdmin).losslessClaim(1),
+        ).to.be.revertedWith('LSS: Already claimed');
+      });
     });
-  });
 
-  describe('when lossless claim is called by not lossless admin two times', () => {
-    it('should revert', async () => {
-      await expect(
-        env.lssGovernance.connect(adr.reporter1).losslessClaim(1),
-      ).to.be.revertedWith('LSS: Must be admin');
+    describe('when lossless claim is called by not lossless admin two times', () => {
```

Comment:
In the test it seems that claim is called only one time?

Reply:
Addressed on 4bf0727